src/api/specimen/GetSpecimenFull.ts

```diff
@@ -47,7 +47,7 @@ const GetSpecimenFull = async (handle: string) => {
                     const annotationIndicator: string = annotation['oa:target']['oa:selector']?.['ods:field'] as string
                         ?? annotation['oa:target']['oa:selector']?.['oa:class'] as string;
 
-                    if (specimenAnnotations[annotation['oa:target']['oa:selector']?.[annotationIndicator] as string]) {
+                    if (annotationIndicator.replace('$.', '') as string in specimenAnnotations) {
```

Comment:
Why do you have the line `annotationIndicator.replace('$.', '')` in the condition as well as in the `else` block (line 53)

Reply:
You have a point, added the replace statement before the conditional to make it globally